src/Exception/TelegramException.php

```diff
@@ -1,17 +1,26 @@
 <?php
-/*
+/**
  * This file is part of the TelegramBot package.
  *
  * (c) Avtandil Kikabidze aka LONGMAN <akalongman@gmail.com>
  *
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
-*/
+ */
+
 namespace Longman\TelegramBot\Exception;
 
+/**
+ * Main exception class used for exception handling
+ */
 class TelegramException extends \Exception
 {
-
+    /**
+     * Exception constructor that writes the exception message to the logfile
+     *
+     * @param string  $message Error message
+     * @param integer $code    Error code
+     */
     public function __construct($message, $code = 0)
```

Comment:
It would be fine to add a method to enable and disable exception log, maybe with a defining a var or a global variable?